components/grid/editing/overview.md

```diff
@@ -449,6 +449,7 @@ There are a few considerations to keep in mind with the CUD operations of the gr
 
 @[template](/_contentTemplates/common/grid-treelist-editing-notes.md#grid-treelist-data-operations-while-editing)
 
+* When editing a master row in a [hierarchy Grid]({%slug components/grid/features/hierarchy%}), the respective `DetailTemplate` will collapse unless you [override the `Equals()` method of the master data item class]({%slug grid-kb-editing-in-hierarchy%}).
```

Comment:
The `## Notes` section of the article has become so long that you could split it into two or even three new articles. My concern here is that if the page visitors don't know exactly what they are looking for, they will never find it.

Reply:
I agree, but we'll leave that for a major revamp that is necessary for other reasons too.